scss/widgets/generic/treeView/_sizes.scss

```diff
@@ -22,11 +22,15 @@ $generic-treeview-border-padding: 1px !default;
 $generic-treeview-loadincicator-offset-top: 8px !default;
 $generic-treeview-loadincicator-offset-left: -1px !default;
 
+$generic-treeview-item-side-padding: 6px !default;
+
+$generic-treeview-custom-expander-icon-offset: null !default;
```

Comment:
Variables with null value are above. Seems it helps us understand which variables will redeclared with new value in each of sizing mode. May be we move it there?